### PR TITLE
285239 - Update params menu

### PR DESCRIPTION
**Pull request by B-Rass**

### CODE REVIEW

File: app/views/contract_types/edit.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <%= render 'layouts/breadcrumb', breadcrumb_variables: {
   t("inventories") => modeles_url,
-  ContractType.model_name.human.pluralize => contract_types_path,
+  ContractType.model_name.human(count: 2) => contract_types_path,
```
Comment (nicolas-brousse) [inline app/views/contract_types/edit.html.erb:3]: @B-Rass why `(count: 2)` over `pluralize`?

File: app/views/layouts/_inventories_dropdown.html.erb [hunk 2]
```diff
@@ -17,17 +17,19 @@
     <li><a href="<%= servers_path %>">Matériels</a></li>
 
     <li role="separator" class="divider"></li>
-
-    <li class="dropdown-header">Gestion du matériel</li>
+    <li class="dropdown-header"><%= t("pages.params_menu.features.title")%></li>
```
Comment (nicolas-brousse) [inline app/views/layouts/_inventories_dropdown.html.erb:20]:
```suggestion
    <li class="dropdown-header"><%= t("inventories_menu.features.title")%></li>
```

@B-Rass suggestion

**Review by nicolas-brousse — CHANGES_REQUESTED** [review]

File: app/views/contract_types/edit.html.erb (hunk 1 above)
Comment (B-Rass) [reply app/views/contract_types/edit.html.erb:3]: pluralize -> `Type de contrats`
count: 2 -> `Types de contrat`

**Review by nicolas-brousse — APPROVED** [review]

**Comment by nicolas-brousse** [comment]
@B-Rass two conflicts to resolve, otherwise LGTM

**Pull request merged**